applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/stress_response_definitions.h

```diff
@@ -48,7 +48,8 @@ namespace Kratos
         MYZ,
         MZX,
         MZY,
-        MZZ
+        MZZ,
+        PK2_TRUSS
```

Comment:
Is this specific to the Truss? => otherwise `PK2` would be a better name

Reply:
I can rename it